### PR TITLE
Supervised Machine Learning Week 1 Documentation

### PR DESCRIPTION
**Pull request by Ghost-141**
This pull request introduces the weekly report for Week 1 of the Supervised Machine Learning module. The notes cover the foundational concepts and introductory algorithms.

## Topics covered in this report ✅

Supervised vs. Unsupervised Learning

Linear Regression:
  
  1. Standard Notation
  
  2. Cost Function
  
  3. Gradient Descent
  
  4. Learning Rate

# Topics still in progress (Week 2) 🛠️ 

1. Multiple Linear Regression

2. Implementation of the Regression Function (using both loop and vectorized forms)

3. Gradient Descent Implementation (vectorized form)

4. Alternatives to Gradient Descent

### CODE REVIEW

**Review by WaterMenon09 — CHANGES_REQUESTED** [review]
Please fix title
